zend_abstract_interface/sandbox/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+add_library(zai_sandbox "${PHP_VERSION_DIRECTORY}/sandbox.c")
+
+target_include_directories(zai_sandbox PUBLIC
+                                       $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
+                                       $<INSTALL_INTERFACE:include>)
+
+target_compile_features(zai_sandbox PUBLIC c_std_99)
+
+target_link_libraries(zai_sandbox PUBLIC "${PHP_LIB}")
+
+set_target_properties(zai_sandbox PROPERTIES
+                                  EXPORT_NAME Sandbox
+                                  VERSION ${PROJECT_VERSION})
```

I prefer that we don't align the initial indents like this. What do you think about this style?

```suggestion
target_include_directories(zai_sandbox
  PUBLIC
    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
    $<INSTALL_INTERFACE:include>)

target_compile_features(zai_sandbox PUBLIC c_std_99)

target_link_libraries(zai_sandbox PUBLIC "${PHP_LIB}")

set_target_properties(zai_sandbox PROPERTIES
  EXPORT_NAME Sandbox
  VERSION ${PROJECT_VERSION})
```

---

Is there a standard formatting for listfiles? All of the examples I've seen so far feel a bit foreign to me. This formatting was inspired by `cmake-format` (see [example](https://github.com/cheshirekow/cmake_format#example)). My preference is just to decided on something that we can keep consistent and add it to the linter in CI. And bonus points if we can just run, `cmake-format -i CMakeLists.txt`, so that I don't have to think about anything at all. :)

---

I'm not aware of a "standard", but I agree that formatting isn't a very big deal if we can run a formatter tool.